student.rb

```diff
@@ -0,0 +1,15 @@
+class Student < Person
+  def initialize(age:, classroom:, name: 'Unknown', parent_permission: true)
+    super(age: age, name: name, parent_permission: parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯(ツ)/¯'
+  end
+
+  def add_classroom(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
```

Comment:
- [ ] The previous reviewer mentioned that we also need a getter/getter for a classroom. Kindly refer to the previous reviewer's comment for your reference.

Reply:
@elisha2kyakpo1  Please could you explain to me more

Reply:
Hi @AbdifatahYasin1, do you know a getter or a setter?

Reply:
I do not want to answer for you, the previous reviewer tried to help with all the answers.

Reply:
yeah

Reply:
Kindly take a screenshot of my review for the student class and circle out the getter or setter for a classroom attribute. I want to know how I can help you.